setup.py

```diff
@@ -3,29 +3,28 @@
 
 Author: Matthew Matl
 """
-import sys
 from setuptools import setup
 
 # load __version__
 exec(open('pyrender/version.py').read())
 
-def get_imageio_dep():
-    if sys.version[0] == "2":
-        return 'imageio<=2.6.1'
-    return 'imageio'
 
 requirements = [
-    'freetype-py',                # For font loading
-    get_imageio_dep(),            # For Image I/O
-    'networkx',                   # For the scene graph
-    'numpy',                      # Numpy
-    'Pillow',                     # For Trimesh texture conversions
-    'pyglet>=1.4.10',             # For the pyglet viewer
-    'PyOpenGL==3.1.0',            # For OpenGL
-#    'PyOpenGL_accelerate==3.1.0', # For OpenGL
-    'scipy',                      # Because of trimesh missing dep
-    'six',                        # For Python 2/3 interop
-    'trimesh',                    # For meshes
+    'freetype-py',                            # For font loading
+    'imageio<=2.6.1; python_version < "3"',   # For Image I/O
+    'imageio<=2.15.0; python_version <= "3.6" and python_version > "3"',
+    'imageio; python_version > "3.6"',
+    'networkx',                               # For the scene graph
+    'numpy<=1.19.5; python_version <= "3.6"', # Numpy
+    'numpy; python_version > "3.6"',
+    'Pillow',                                 # For Trimesh texture conversions
+    'pyglet>=1.4.10',                         # For the pyglet viewer
+    'PyOpenGL==3.1.0',                        # For OpenGL
+#   'PyOpenGL_accelerate==3.1.0',             # For OpenGL
+    'scipy<=1.5.4; python_version <= "3.6"',  # Because of trimesh missing dep
+    'scipy; python_version > "3.6"',          
+    'six',                                    # For Python 2/3 interop
+    'trimesh<=3.9.32',                        # For meshes
```

Comment:
I don't know what change there was in trimesh in 3.9.33 but the following 2 lines of the test fail so I restricted the version: 


https://github.com/mmatl/pyrender/blob/4a289a6205c5baa623cd0e7da1be3d898bcbc4da/tests/unit/test_meshes.py#L121-L122